### PR TITLE
[NS2.1] More types and members

### PR DESCRIPTION
**Pull request by EgorBo**
Contributes to https://github.com/mono/mono/issues/11430

Most changes are in mono/corefx PR I am preparing now.

### CODE REVIEW

File: mcs/class/corlib/corlib.dll.sources [hunk 1]
```diff
@@ -189,7 +189,7 @@ System.Diagnostics.SymbolStore/ISymbolScope.cs
 System.Diagnostics.SymbolStore/ISymbolVariable.cs
 System.Diagnostics.SymbolStore/ISymbolWriter.cs
 System.Diagnostics.SymbolStore/SymAddressKind.cs
-System.Diagnostics.SymbolStore/SymbolToken.cs
```
Comment (marek-safar) [inline mcs/class/corlib/corlib.dll.sources]: Please remove the file

**Comment by marek-safar** [comment]
build

**Comment by EgorBo** [comment]
@monojenkins build failed

**Comment by marek-safar** [comment]
@monojenkins commit csproj and apidiff

**Comment by lambdageek** [comment]
This broke an acceptance test from coreclr.  See https://github.com/mono/mono/issues/12242

**Pull request merged**